src/app/modules/modal-shell/modal-shell.component.ts

```diff
@@ -9,7 +9,7 @@ import {
 @Component({
   selector: 'app-modal-shell',
   templateUrl: './modal-shell.component.html',
-  styleUrls: ['./modal-shell.component.css']
+  styleUrls: ['./modal-shell.component.scss']
```

Comment:
oops 😱

Reply:
I'm confused... what's `oops`? That we weren't using `scss` before? Looking around at other components, doesn't appear we are? @mcelestini

Reply:
exactly. we weren't pulling in the right style file

Reply:
Oh, so I changed the original file from css to scss so that I could nest some styles...let me know if that's not ok.

Reply:
ahhhh... whoops my bad. Missed that. Yeah that's okay 👍